cloud-sql/postgres/app.yaml

```diff
@@ -2,5 +2,11 @@
 runtime: aspnetcore
 env: flex
 
+env_variables:

```

Comment:
I understand these are samples and that this change actually is meant to make the samples more similar across language. But having a DB password in an environment variable is really not a good practice to show/follow.
I would prefer that you'd reconsider and leave the connections string details on [configuration files](https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/connection-strings-and-configuration-files), which is common in .NET and can be encrypted. Else, do please add a comment on the readme, saying that this is done for the purposes of samples only and that storing a plain text DB password in an environment variable is not secure.

Reply:
And this comment applies to both the postgres and the sql sample.

Reply:
@shubha-rajan Can you make sure [we add the secret manager message](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/948696d8a43f7d6b489a574ae356971471ebcdc7/cloud-sql/mysql/sqlalchemy/main.py#L99) about plaintext secrets around the env vars? 

We should make sure this is on our checklist of things when trying to standardize these samples as well